samples/nrf9160/modem_shell/README.rst

```diff
@@ -379,6 +379,18 @@ Building and running
 
 .. include:: /includes/build_and_run_nrf9160.txt
 
+DK buttons
+==========
+
+The buttons have the following functions:
+
+Button 1:
+    * Pressing of the button: raising a kill/abort signal. I.e. long enough press of the button will kill/abort all supported running commands.
+    * Releasing of the button: resetting a kill/abort signal.
+
+Button 2:
+    * Enabling / disabling the UART of the MoSh.
```

Comment:
I would suggest changing this a bit, because the implementation disables both UARTs. Also I think it's good to mention the use case, because it might not be obvious.
```suggestion
    * Enabling / disabling the UARTs for power consumption measurements.
```

Reply:
Updated.